Dockerfile

```diff
@@ -2,11 +2,12 @@ FROM golang:alpine
 
 MAINTAINER siddontang
 
-COPY . /go/src/github.com/siddontang/go-mysql-elasticsearch
+ADD . /go/src/github.com/siddontang/go-mysql-elasticsearch
 
 RUN cd /go/src/github.com/siddontang/go-mysql-elasticsearch/ && \
     go build -o bin/go-mysql-elasticsearch ./cmd/go-mysql-elasticsearch && \
     cp -f ./bin/go-mysql-elasticsearch /go/bin/go-mysql-elasticsearch
 
-ENTRYPOINT ["go-mysql-elasticsearch"]
+RUN apk add --no-cache tini mariadb-client
```

Comment:
can we move it up before L5? So we can still use the docker cache even we change the source.

Reply:
OK!